src/firefly/java/edu/caltech/ipac/firefly/server/query/StatisticsProcessor.java

```diff
@@ -66,8 +66,13 @@ protected DataGroup fetchData(TableServerRequest treq, DbAdapter dbAdapter) thro
                 row.setDataElement(columns[0], cname);
                 row.setDataElement(columns[1], desc);
                 row.setDataElement(columns[2], units);
+
+                String maxExp = col.isWholeNumber() ?
+                        "max(\"%1$s\") as \"%1$s_max\"".formatted(cname) :
+                        "max(if(\"%1$s\" = 'nan', null, \"%1$s\")) as \"%1$s_max\"".formatted(cname);    // for duckdb, NaN is greater than any other floating point number; therefore, we will exclude it by setting to null
+
```

Comment:
This is interesting about `NaN` since in java script a `NaN` comparison is always false.

Reply:
Yes, this is specific to DuckDB.

```
... DuckDB follows the IEEE 754 specification regarding these special values, with two exceptions:

- NaN compares equal to NaN and greater than any other floating point number.
- Some floating point functions, like sqrt / sin / asin throw errors rather than return NaN for values outside their ranges of definition.
```